### ISSUE TITLE
Deprecate or remove NDS/DDS/NDS?

### ISSUE THREAD

**Issue by Ptival** [GaloisInc/flexdis86#50]
Those VEX specifiers no longer appear in the Intel manuals (and never appeared in AMD?), as they are subsumed by the operand encoding tables, which we also specify in `<opr></opr>`.

<img width="692" alt="image" src="https://github.com/GaloisInc/flexdis86/assets/478606/793212be-a0e7-4827-b262-34e5dfd817f7">

Yet we are syntactically forced to specify them in `optable.xml`.

I don't see anywhere where we're using them, so I'd be in favor of just dropping them entirely.

**Comment by RyanGlScott** [comment]
Yes, I agree. If we aren't using these prefixes, we shouldn't bother specifying them.
